spec/views/comments/show.html.erb_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+RSpec.describe "comments/show", type: :view do
+  before(:each) do
+    @comment = assign(:comment, Comment.create!(
+      :body => "Body"
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)